src/tests/components/ForecastSummary.test.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: 1525046400000,
+    description: "Stub description",
+    icon: "800",
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+    onSelect: () => {},
```

Comment:
In this instance it doesn't matter that much because you're tests aren't using the `onSelect` function, but it's still good practice to use mocked functions like:
`onSelect: Jest.fn()`